src/Permissions/Backend/App.vala

```diff
@@ -119,10 +119,24 @@ public class Permissions.Backend.App : GLib.Object {
     }
 
     private string get_overrides_path () {
-        return GLib.Path.build_path (
+        var overrides_folder_path = GLib.Path.build_path (
             GLib.Path.DIR_SEPARATOR_S,
             AppManager.get_default ().user_installation_path,
-            "overrides",
+            "overrides"
+        );
+
+        var overrides_folder = File.new_for_path (overrides_folder_path);
+        if (!overrides_folder.query_exists ()) {
+            try {
+                overrides_folder.make_directory ();
+            } catch (Error e) {
+                critical ("Couldn't create overrides folder: %s", e.message);
+            }
+        }
+
+        return GLib.Path.build_path (
```

Comment:
There is a possibility that the path will point to a non-existent file - admittedly a lower possibility than before so still worthwhile. Not sure what correct return is if there is an error creating the folder.